tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -227,3 +227,49 @@ def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, r
         {'organization-id': loadbalancer_setup['module_org'].id}
     )
     assert rhel7_contenthost.hostname in [host['name'] for host in hosts]
+
+
+@pytest.mark.tier1
+def test_client_register_through_lb(
+    loadbalancer_setup, rhel7_contenthost, module_target_sat, module_org, module_ak, module_location
+):
+    """Register the client through loadbalancer
+
+    :id: c7e47d61-167b-4fc2-8d1a-d9a64350fdc4
+
+    :Steps:
+
+        1. Try package installation
+        2. run `subscription-manager register --org=Your_Organization \
+            --activationkey=Your_Activation_Key \
+            --serverurl=https://loadbalancer.example.com:8443/rhsm \
+            --baseurl=https://loadbalancer.example.com/pulp/content`
```

Comment:
I think we're not manually registering here, so could you update the steps here?